core/keyserver/epochs.go

```diff
@@ -134,48 +164,28 @@ func (*Server) ListMutationsStream(in *pb.ListMutationsRequest, stream pb.KeyTra
 	return status.Errorf(codes.Unimplemented, "ListMutationStream is unimplemented")
 }
 
-func (s *Server) logProofs(ctx context.Context, logID, firstTreeSize int64, epoch int64) (
-	*tpb.GetLatestSignedLogRootResponse,
-	*tpb.GetConsistencyProofResponse,
-	*tpb.GetInclusionProofResponse,
-	error) {
-	// Lookup log and map info.
-	logRoot, err := s.tlog.GetLatestSignedLogRoot(ctx,
-		&tpb.GetLatestSignedLogRootRequest{
-			LogId: logID,
-		})
+// logProofs returns the proofs for a given epoch.
+func (s *Server) logProofs(ctx context.Context, d *domain.Domain, firstTreeSize int64, epoch int64) (*tpb.SignedLogRoot, *tpb.Proof, *tpb.Proof, error) {
```

Comment:
Wonder if the proofs for an epoch could be wrapped into a single object for a cleaner signature.

Reply:
That's a bit better.